insights/parsers/hammer_compute_resource_list.py

```diff
@@ -0,0 +1,56 @@
+"""
+HammerComputeResourceList - command ``hammer --interactive 0 --output json compute-resource list``
+===================================================
+
+This parser reads the compute resource list of a Satellite server using hammer in JSON format.
+It relies on the root user running the command and hammer to authenticate based on the hammer configuration file.
+If the command is unable to authenticate then no compute resources will be shown and an erro flag will be recorded in the parser
+
+Sample output from the ``hammer --interactive 0 --output json compute-resource list`` command::
+
+    [
+  {
+    "Id": 1,
+    "Name": "kvm-server",
+    "Provider": "Libvirt"
+  },
+  {
+    "Id": 4,
+    "Name": "VMWARE",
+    "Provider": "VMware"
+  },
+  {
+    "Id": 3,
+    "Name": "vmware67",
+    "Provider": "VMware"
+  }
+]
+
+Examples:
+
+    >>> type(hammer_compute_resources_list)
+    <class 'insights.parsers.hammer_compute_resource_list.HammerComputeResourceList'>
+"""
+
+from insights import parser, CommandParser
+from insights.parsers import SkipException
+from insights.specs import Specs
+import json
+
+
+@parser(Specs.hammer_compute_resource_list)
+class HammerComputeResourceList(CommandParser):
```

Comment:
There is a `JSONParser` that can be used as one of its base classes. Would you please change to use it?

https://github.com/RedHatInsights/insights-core/blob/bf3527ef533ba14dbdbc60edda77d38232b9692b/insights/core/__init__.py#L744-L756

Here is an example:

https://github.com/RedHatInsights/insights-core/blob/bf3527ef533ba14dbdbc60edda77d38232b9692b/insights/parsers/ceph_cmd_json_parsing.py#L44-L56